bedita-app/models/behaviors/build_filter.php

```diff
@@ -1183,9 +1183,10 @@ protected function allowed_to_userFilter($s, $e, $userid) {
 
                 if (!empty($forbiddenPub) || !empty($forbiddenSection)) {
 
-                    $query = "SELECT {$s}Tree{$e}.{$s}id{$e}
+                    $query = "SELECT DISTINCT {$s}Tree{$e}.{$s}id{$e}
                              FROM {$s}trees{$e} AS {$s}Tree{$e}
-                             WHERE {$s}BEObject{$e}.{$s}id{$e} = {$s}Tree{$e}.{$s}id{$e}";
+                             INNER JOIN {$s}objects{$e} AS {$s}BEObject{$e} ON {$s}BEObject{$e}.{$s}id{$e} = {$s}Tree{$e}.{$s}id{$e}
+                             WHERE 1=1";
```

Comment:
```suggestion
                             INNER JOIN {$s}objects{$e} AS {$s}BEObject{$e} ON {$s}BEObject{$e}.{$s}id{$e} = {$s}Tree{$e}.{$s}id{$e}";
```

Should work without `WHERE` clause too…